### PR TITLE
Massage OpenAPI Spec for Code Generation

### PR DESCRIPTION
**Pull request by TeisJayaswal**
In order to generate code using fern, the OpenAPI spec must pass fern check. Here's a summary of the changes we made to get to green:
- Reformatted a couple types 
- Renamed types beginning with a number and updated the references

### CODE REVIEW

File: fern/api/openapi/openapi.yaml [hunk 1]
```diff
@@ -11892,8 +11892,9 @@ components:
     BankAccountPseResponse:
       type: object
       title: Colombia 🇨🇴 PSE
-      oneOf:
-        - $ref: '#/components/schemas/BankAccountBusinessPse'
+      properties:
+        type:
+          $ref: '#/components/schemas/BankAccountBusinessPse'
```
Comment (dsinghvi) [inline fern/api/openapi/openapi.yaml:11897]: what was the error when this was `oneOf` ?

File: fern/api/openapi/openapi.yaml [hunk 2]
```diff
@@ -13169,9 +13170,11 @@ components:
...
       title: Colombia 🇨🇴 PSE
       description: Object containing the payer's bank account information.
-      oneOf:
-        - $ref: '#/components/schemas/TransactionBankAccountBodyPse'
-        - $ref: '#/components/schemas/BankAccountInformationPse'
+      properties:
+        type:
+          oneOf:
+            - $ref: '#/components/schemas/TransactionBankAccountBodyPse'
+            - $ref: '#/components/schemas/BankAccountInformationPse'
```
Comment (dsinghvi) [inline fern/api/openapi/openapi.yaml:13177]: why did you add the oneOf to a property called `type` ?

File: fern/api/openapi/openapi.yaml (hunk 1 above)
Comment (TeisJayaswal) [reply fern/api/openapi/openapi.yaml:11897]: It was failing to convert: [api]: Failed to convert schema breadcrumbs=["BankAccountPseResponse"] value={"type":"object","title":"Colombia 🇨🇴 PSE","oneOf":[{"$ref":"#/components/schemas/BankAccountBusinessPse"}]}
